dataproc/create_cluster.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_cluster(project_id, region, cluster_name):
+    # [START create_cluster]
```

Comment:
Does dataproc have the product_name in each region tag?

`# [START dataproc_create_cluster]`

Mostly so these show up in sample tracker